src/components/Tags/__tests___/Tag.test.tsx

```diff
@@ -37,24 +31,34 @@ describe('Tag', () => {
       color: colors.solidFaintLight,
       fontSize: typography.fontSizeMini,
       fontFamily: typography.fontFamilySupport,
-      fontWeight: typography.fontWeightMedium.toString() as FontWeightType,
+      fontWeight: `${typography.fontWeightMedium}`,
     });
   });
 
-  it('renders correctly with custom props', () => {
+  it('renders tag correctly with an icon', () => {
+    const { getByTestId } = render(<Tag text="some tag" testID="TagID" icon="Clock" />);
+
+    const icon = getByTestId('TagID.Icon');
+
+    expect(icon).toBeTruthy();
+  });
+});
+
+describe.each(Object.keys(TAG_PROPERTIES))('renders correctly with custom variant %s', (variant) => {
+  it.each(Object.keys(TAG_PROPERTIES[variant as TagVariant]))('renders correctly with custom type %s', (type) => {
```

Comment:
💅 It's not a custom type, is predefined
```suggestion
  it.each(Object.keys(TAG_PROPERTIES[variant as TagVariant]))('renders correctly with type %s', (type) => {
```

Reply:
Fixed in [6d26c52](https://github.com/magnetis/astro-native/pull/791/commits/6d26c5202fd3663b691c44c889a373d7f4ebd335)